src/ansys/fluent/core/exceptions.py

```diff
@@ -0,0 +1,11 @@
+"""Custom common higher level exceptions."""
+
+
+class DisallowedValuesError(ValueError):
+    def __init__(self, error):
+        super().__init__(error)
```

Comment:
I believe that overloading the `__init__` dunder method is unnecessary, the default inheritance behavior already achieves the same:
```suggestion
class DisallowedValuesError(ValueError):
    pass
```
Same for the other exception definitions

Reply:
Right, It will also remove these 2 lines and code will look cleaner. Thank you.